src/client/components/TotalPriceCard/TotalPriceCard.styles.css

```
@@ -0,0 +1,28 @@
+.wrapper {
+  width: 18rem;
+  height: 10rem;
+  border: 1px solid lightgray;
+  border-radius: 0.5rem;
+}
+
+.wrapper:hover {
+  border: 1px solid black;
+}
+
+.wrapper-content {
+  padding: 0 1rem;
+}
+
+.row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.text-right {
+  text-align: right;
+}
+
+h3 {
```

I think you can add your first classname here. Because only h3 can affect other CSS files.
```suggestion
.wrapper h3 {
```

---

👍